### DISCUSSION TITLE
Animation is choppy in Low Power Mode

### DISCUSSION THREAD

**Discussion by jrosen081** [airbnb/lottie-ios#1867] [General]
This only happens when on the core animation rendering engine. When the rendering engine is main thread, it works as expected.

## Which Version of Lottie are you using?
Lottie 3.4.3

## Expected Behavior 
(this is the behavior when not in low power mode)
https://user-images.githubusercontent.com/20172570/207895318-1c679a71-de76-4f00-83d7-c151c7d6ef3c.MP4

## Actual Behavior
(this is the behavior when in low power mode)
https://user-images.githubusercontent.com/20172570/207895423-592411ea-f66b-4e5b-90bb-bc8954e9763e.MP4

## Animation JSON
[Choppy_Animation.json.zip](https://github.com/airbnb/lottie-ios/files/10237935/Choppy_Animation.json.zip)

**Comment by calda** [comment]
This sounds like an expected behavior of iOS system animations, which are being intentionally throttled by the OS to reduce power consumption when the device is in low power mode.
